Library/Homebrew/dev-cmd/boneyard-formula-pr.rb

```diff
@@ -0,0 +1,130 @@
+# Creates a pull request to boneyard a formula.
+#
+# Usage: brew boneyard-formula-pr [options...] <formula-name>
+#
+# Options:
+#   --dry-run:  Print what would be done rather than doing it.
+
+require "formula"
+require "utils/json"
+require "fileutils"
+
+begin
+  require "json"
+rescue LoadError
+  puts "Homebrew does not provide Ruby dependencies; install with:"
+  puts "  gem install json"
+  odie "Dependency json is not installed."
+end
+
+module Homebrew
+  def boneyard_formula_pr
+    formula = ARGV.formulae.first
+    odie "No formula found!" unless formula
+
+    formula_path = formula.path
+    formula_file = "#{formula.name}.rb"
+    bottle_block = File.read(formula_path).include? "  bottle do"
+    boneyard_tap = Tap.fetch("homebrew", "boneyard")
+    if ARGV.dry_run?
+      puts "#{HOMEBREW_BREW_FILE} update"
+      puts "#{HOMEBREW_BREW_FILE} tap homebrew/boneyard"
```

Comment:
Just in case this was lost in a refresh of the PR: I'd stick with just `brew` in the output in dry-run mode. Looks cleaner to me. (Same for similar output in dry-run mode further below.)